src/workOrder/controller.js

```diff
@@ -241,6 +251,8 @@ async function getAllWithPackedQties(showFulfilled) {
     const d = await ShopQueue.aggregate(agg);
     const data = d?.[0]?.activeWorkOrders;
 
+    console.debug(data.filter(x => x.orderNumber === 'WAABI1006'));
```

Comment:
@jarrilla did you mean to leave this in here?

Reply:
Nope. Accidentally left it in there after i was using that order number as a good sample for bad data. Thanks for catching it